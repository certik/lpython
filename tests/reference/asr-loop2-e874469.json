{
    "basename": "asr-loop2-e874469",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/loop2.py",
    "infile_hash": "0f1af9370fbae5de0ed9e612e933d7745598fc9c74b036496f4ae8ca",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-loop2-e874469.stdout",
    "stdout_hash": "e310c8c30a336eecd524851e7d6664a09f9b3910b7d641c7fea48f8d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}