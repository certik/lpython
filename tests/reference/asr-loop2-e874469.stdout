(TranslationUnit (SymbolTable 1 {exit: (ExternalSymbol 1 exit 3 exit sys [] exit Public), main_program: (Program (SymbolTable 6 {}) main_program [] []), sys: (Module (SymbolTable 3 {exit: (Subroutine (SymbolTable 4 {error_code: (Variable 4 error_code In () () Default (Integer 4 []) Source Public Required .false.)}) exit [(Var 4 error_code)] [(Stop (Var 4 error_code))] Source Public Implementation () .false. .false.)}) sys [] .false. .false.), test_for: (Subroutine (SymbolTable 5 {i: (Variable 5 i Local () () Default (Integer 4 []) Source Public Required .false.)}) test_for [] [(DoLoop ((Var 5 i) (ConstantInteger 0 (Integer 4 [])) (BinOp (ConstantInteger 10 (Integer 4 [])) Sub (ConstantInteger 1 (Integer 4 [])) (Integer 4 []) (ConstantInteger 9 (Integer 4 [])) ()) (ConstantInteger 1 (Integer 4 []))) [(If (Compare (Var 5 i) Eq (ConstantInteger 0 (Integer 4 [])) (Logical 4 []) () ()) [(Cycle)] []) (If (Compare (Var 5 i) Gt (ConstantInteger 5 (Integer 4 [])) (Logical 4 []) () ()) [(Exit)] []) (If (Compare (Var 5 i) Eq (ConstantInteger 3 (Integer 4 [])) (Logical 4 []) () ()) [(Stop ())] [])]) (SubroutineCall 1 exit () [(ConstantInteger 0 (Integer 4 []))] ())] Source Public Implementation () .false. .false.)}) [])
